Test scripts/Ilse/bulk-poli-temp-to-word.vbs

```diff
@@ -73,17 +94,33 @@ Do
         err_msg = ""
         dialog Dialog1
         cancel_without_confirmation
-        If ButtonPressed = select_a_file_button then call file_selection_system_dialog(file_selection_path, ".xlsx")
-        If trim(file_selection_path) = "" then err_msg = err_msg & vbcr & "* Select a file to continue."
-        If err_msg <> "" Then MsgBox err_msg
+        'Initial POLI TEMP List Excel file
+        If ButtonPressed = select_initial_file_button then call file_selection_system_dialog(file_selection_path, ".xlsx")
+        If trim(file_selection_path) = "" then err_msg = err_msg & vbcr & "* Select the initial POLI TEMP List file to continue."
+        'Folder selection where the Word Docs are going to live
+        If ButtonPressed = select_save_folder_button then call select_folder(root_file_path)
```

Comment:
Not sure if the user would ever need to navigate to a different folder but the "Browse for Folder" dialog appears to be limited in that I couldn't figure out how to navigate to a different folder (either moving up or across folder hierarchy).